tests/basic/tests.py

```diff
@@ -357,6 +357,24 @@ def test_hash(self):
             # hash)
             hash(Article())
 
+    def test_missing_hash_not_inherited(self):
+        class NoHash(models.Model):
+            def __eq__(self, other):
+                return super.__eq__(other)
+
+        with self.assertRaisesMessage(TypeError, "unhashable type: 'NoHash'"):
+            hash(NoHash(id=1))
```

Comment:
```suggestion
            hash(NoHash())
```

Reply:
Models without ids can’t be hashed, which does raise a different error, but with the id is the minimum valid instance that can be hashed. 

As such, I put it in. 

Happy to go your way if you still prefer?

Reply:
Thanks for explanation, I just want to minimize test, so your version with `id` looks good :+1: